caffeinated/admin/extend/events/Extend_Events_Admin_Page.core.php

```diff
@@ -1269,4 +1301,115 @@ protected function _delete_the_ticket($TKT_ID)
         $tkt->delete_related_permanently('Price');
         return $tkt->delete_permanently();
     }
+
+
+    /**
+     * @param array $default_event_settings_form_subsections
+     * @return array
+     * @since $VID:$
+     */
+    public function advancedEditorAdminFormSection(array $default_event_settings_form_subsections)
+    {
+        return [
+            'advanced_editor_header' => new EE_Form_Section_HTML(
+                EEH_HTML::div(
+                    EEH_HTML::div(
+                        EEH_HTML::h2(
+                            esc_html__('New Feature', 'event_espresso'),
+                            '',
+                            'ee-admin-settings-hdr ee-new-flag'
+                        ),
+                        '',
+                        'ee-new-flag-wrap'
+                    ),
+                    '',
+                    'ee-new-flag-shadow',
+                    'margin: -25px 0 10px;'
+                )
+            ),
+            'use_advanced_editor'         => new EE_Select_Input(
+                apply_filters(
+                    'FHEE__Events_Admin_Page___default_event_settings_form__advanced_editor_input_options',
+                    [
+                        esc_html__('Legacy Editor', 'event_espresso'),
+                        esc_html__('Advanced Editor', 'event_espresso'),
+                    ]
+                ),
+                [
+                    'default'         => $this->admin_config->useAdvancedEditor(),
+                    'html_label_text' => esc_html__('Activate Advanced Editor?', 'event_espresso'),
+                    'html_help_text'  => sprintf(
+                        esc_html__(
+                            'Controls whether the Event Espresso Event Editor continues to use the existing legacy editor that functions like the typical older WordPress admin you are used to,%1$sor uses the new Advanced Editor with a more powerful and easier to use interface. This may be automatically turned on in order to utilize advanced features from new addons.',
+                            'event_espresso'
```

Comment:
The "this may be automatically turned on in order to use advanced features" is good text here for when the REM add-on is active.  I wonder if it'd be good to add an hook somewhere here so REM can _disable_ this input and add it's own text explaining why the option is disabled (so that the user knows why they can't change it and why it is forced on)?

Reply:
Did you see this comment I added ^^?

Reply:
oops mistook which block of text this comment was made on, and thought it applied to one of the options that has been removed. Thought it would be obvious why I didn't respond to a comment regarding something that was removed, but that was not the case. My apologies. I've added a filter for the input settings as well.